pkg/controller/common/common.go

```diff
@@ -312,20 +323,32 @@ func (m GroupKindMap) Get(gk schema.GroupKind) *dynamicdiscovery.APIResource {
 	return m[gk]
 }
 
-type InformerMap map[schema.GroupVersionResource]*dynamicinformer.ResourceInformer
+type InformerMap map[schema.GroupVersionResource]informers.GenericInformer
 
-func (m InformerMap) Set(gvr schema.GroupVersionResource, informer *dynamicinformer.ResourceInformer) {
+func (m InformerMap) Set(gvr schema.GroupVersionResource, informer informers.GenericInformer) {
 	m[gvr] = informer
 }
 
-func (m InformerMap) Get(gvr schema.GroupVersionResource) *dynamicinformer.ResourceInformer {
+func (m InformerMap) Get(gvr schema.GroupVersionResource) informers.GenericInformer {
 	return m[gvr]
 }
 
 // GetObject return object via Lister from given informer, namespaced or not.
-func GetObject(informer *dynamicinformer.ResourceInformer, namespace, name string) (*unstructured.Unstructured, error) {
+func GetObject(inf informers.GenericInformer, namespace, name string) (runtime.Object, error) {
```

Comment:
I have issue with typing here, as `runtime.Object` returned by `informers.GenericInformer` is not what I want here (desired is `unstructured.Unstructured`)